kotlin-audio/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
     id 'maven-publish'
 }
 
-def versionNumber = '0.1.34'
+def versionNumber = '1.0'
```

Comment:
After this is merged I think KotlinAudio is ready for a 1.0 release. So also putting this here :)